lib/cc/services/github_pull_requests.rb

```diff
@@ -93,27 +86,34 @@ def update_status_pending
   end
 
   def update_status(state, description)
-    params = {
-      state:       state,
-      description: description,
-      target_url:  @payload["details_url"],
-      context:     "codeclimate"
-    }
-    service_post(status_url, params.to_json)
+    if config.update_status
+      params = {
+        state:       state,
+        description: description,
+        target_url:  @payload["details_url"],
+        context:     "codeclimate"
+      }
+      @response = service_post(status_url, params.to_json)
+    end
   end
 
   def add_comment
-    if !comment_present?
-      body = {
-        body: COMMENT_BODY % @payload["compare_url"]
-      }.to_json
-
-      service_post(comments_url, body) do |response|
-        doc = JSON.parse(response.body)
-        { id: doc["id"] }
+    if config.add_comment
+      if !comment_present?
+        body = {
+          body: COMMENT_BODY % @payload["compare_url"]
+        }.to_json
+
+        @response = service_post(comments_url, body) do |response|
+          doc = JSON.parse(response.body)
+          { id: doc["id"] }
+        end
+      else
+        @response = {
+          ok: false,
```

Comment:
That's also the [current behavior](https://github.com/codeclimate/codeclimate-services/blob/4aa7dfaa1a09438e41c27afa4bcebbe8ea3c9e58/lib/cc/services/github_pull_requests.rb#L116). If we're to change it, let's do it in another PR/card.

But yeah, I agree with you. It used to be the case where the return value may/may not matter, but now it does matter.